packages/aksara-icons/src/media/carousel/IconCarouselDefault.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { BaseIconProps, iconDefaultProps } from '../../utils/types';
+
+const IconCarouselDefault: React.FC<BaseIconProps> = ({ size, fill, ...props }) => (
```

Comment:
```suggestion
const IconCarousel: React.FC<BaseIconProps> = ({ size, fill, ...props }) => (
```

Default-nya dihapus aja di semua nama variabel icon. @fahmi-idris

Reply:
olrait